specs/Alert_Notifications.adoc

```diff
@@ -0,0 +1,186 @@
+
```

Comment:
This spec could be referenced from https://github.com/Tendrl/specifications/pull/198 ?